fs/entry.go

```diff
@@ -58,6 +58,7 @@ type Directory interface {
 	Entry
 	Child(ctx context.Context, name string) (Entry, error)
 	IterateEntries(ctx context.Context, cb func(context.Context, Entry) error) error
+	MultipleIterations() bool
```

Comment:
A comment on what this means would be helpful

Reply:
done